handbook/people-ops/hiring/reference_check_questions.md

```diff
@@ -12,11 +12,9 @@ Requested references will typically be a former manager(s) and/or colleague(s) a
 
 ## Reference call
 
-People Ops will reach out to each reference to schedule calls. People Ops will conduct the reference calls with the hiring manager present to address any potential concerns or follow-up questions.
+People Ops will coordinate a calls with references and hiring managers. People Ops will conduct the reference calls with the hiring manager present to address any potential concerns or follow-up questions.
```

Comment:
```suggestion
People Ops will coordinate calls with references and hiring managers. People Ops will conduct the reference calls with the hiring manager present to address any potential concerns or follow-up questions.
```

Reply:
Just to check, who is "People Ops" here? The recruiter specifically? Or is this a central People Ops function?

Reply:
Edited to specify that it should be the recruiter.